tests/test_formatter/test_formatter_output.py

```diff
@@ -115,3 +115,32 @@ def test_formatter_correct(snapshot, cli_options, output):
         _safe_output(stdout),
         'formatter_correct_{0}'.format(output),
     )
+
+
+def test_ipynb(snapshot):
+    """All correct code should not raise any violations and no output."""
+    filename = './tests/fixtures/notebook.ipynb'
+    cli_options = ['--extend-ignore', 'NIP102,D100']
```

Comment:
Can you please write a comment on why do we need this, so we would not forget about it later.